src/utils/timeFromNow.jsx

```diff
@@ -106,7 +107,7 @@ function TimeFromNow(props) {
     moment.relativeTimeThreshold('m', defaultThreshold_m);
     moment.relativeTimeThreshold('h', defaultThreshold_h);
     moment.relativeTimeThreshold('d', defaultThreshold_d);
-    moment.relativeTimeThreshold('M', defaultThreshold_M);
+    moment.relativeTimeThreshold('m', defaultThreshold_M);
```

Comment:
Sorry I didn't flag this previously, but I think it is also wrong from the prior bad find and replace.
I really think we should consider my suggestion to rename these variables to spell out the time unit portion.

Reply:
Fixing.

Reply:
Seems to be correct as of 9aa7069.  Resolving.